kong/tools/queue_schema.lua

```diff
@@ -49,5 +49,13 @@ return Schema.define {
       between = { 0.001, 1000000 }, -- effectively unlimited maximum
       description = "Maximum time in seconds between retries, caps exponential backoff."
     } },
+    { concurrency = {
+      type = "integer",
+      default = 1,
+      between = { 0, 1 },
+      required = true,
```

Comment:
should we omit `required = true`, to be consistent with all the other fields? Probably it makes no difference in practice since we're providing a default but it feels safer.

Reply:
(I usually prefer providing both `required = true` and `default = x` for primitive values, as the `null` is meaningless. Removing `required = true` means it accepts null value (vai admin API), which may cause unexpected behavior in the proxy handler. )

Removing `required = true` to keep code consistent.